proxstar/templates/list_vms.html

```diff
@@ -3,11 +3,11 @@
 
 <div class="container">
     <div class="row">
-        {% if user['rtp'] and rtp_view != True %}
+        {% if user_view %}
```

Comment:
Does this make the rtp view the default now? Have you tested this with the normal user `testing` account? Better not be vulnerable 😤

Reply:
If user_view, the user must be RTP. If not they get 403. Same with the pool page. This is checked in flask